analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/HashesShouldHaveUnpredictableSaltTest.cs

```diff
@@ -34,7 +35,7 @@ public class HashesShouldHaveUnpredictableSaltTest
         public void HashesShouldHaveUnpredictableSalt_CS() =>
             OldVerifier.VerifyAnalyzer(
                 @"TestCases\SymbolicExecution\Sonar\HashesShouldHaveUnpredictableSalt.cs",
-                new SymbolicExecutionRunner(),
+                new SymbolicExecutionRunner(AnalyzerConfiguration.AlwaysEnabledWithSonarCfg),
```

Comment:
This change should not be needed. From `any other random sonar rule` point of view, the runner should run under its default behavior and we should not need to update these UTs.
We should update these when we migrate the rule.

Reply:
This applies to all tests except S2259